### PR TITLE
Rename `ancestor_id_history` to `log_user_id_history`

### PR DESCRIPTION
**Pull request by rockapot**

### CODE REVIEW

File: proto/event/event.proto [hunk 1]
```diff
@@ -541,7 +541,7 @@ message AncestorIdHistoryItem {
 // Next ID = 7.
 message AncestorIdHistory {
   // Window of latest log user ids.
-  repeated AncestorIdHistoryItem ancestor_id_history = 1;
+  repeated AncestorIdHistoryItem log_user_id_history = 1;
```
Comment (danbosnichill) [inline proto/event/event.proto:544]: I'm going to assume nothing serializes this as JSON currently.  This is fine.

**Review by danbosnichill — APPROVED** [review]

**Review by danbosnichill — CHANGES_REQUESTED** [review]
Actually, you should do this in the internal schema first.

**Review by andrewdyates — CHANGES_REQUESTED** [review]
internal-schema?

**Comment by rockapot** [comment]
> Actually, you should do this in the internal schema first.

Oops. Yes, of course. I'll send a new PR.

**Pull request closed without merging**